_data/students.yml

```diff
@@ -90,17 +90,22 @@
   github: Jerry2001
   facebook: ngohongphuc2001
 
+- name: Tushar Kumar
+  country: India
+  github: awesometushar000
+  image: tushar.jpg
+  twitter: meets2tushar
+
 - name: Aryan Kedare
   country: India
   github: aryankedare
   image: aryankedare.jpg
   twitter: KedareAryan
-  
+
 - name: Samyak Jain
   github: samyakbvs
   image: samyak.png
   twitter: Samyakbvs
-  facebook: Samyak Jain 
```

Comment:
Please add this back, don't remove details of other students. Thanks.

Reply:
Done 👍